dwave/cloud/solver.py

```diff
@@ -306,11 +309,14 @@ def sample_qubo(self, qubo, offset=0, **params):
             offset (optional, default=0):
                 Constant offset applied to the model.
 
+            label (str, optional):
+                Problem label.
```

Comment:
Just wondering whether it's worth the additional length to clarify the optionality of this parameter and its intention as an aid for user identification rather than a system need, for example something like, "Problem label you can optionally tag submissions with for ease of identification."